sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -216,6 +255,9 @@ private[sql] object ParquetFilters {
       case sources.IsNotNull(name) =>
         makeNotEq.lift(dataTypeOf(name)).map(_(name, null))
 
+      case sources.In(name, values) =>
+        makeInSet.lift(dataTypeOf(name)).map(_(name, values.toSet))
+
```

Comment:
Thanks for adding this back :) I think this is worth a separate JIRA ticket to track. Would you mind to open one and add the ID to the PR title?